package/MDAnalysis/analysis/hbonds/hbond_autocorrel.py

```diff
@@ -122,22 +136,41 @@
 Examples
 --------
 
+This example finds the continuous hydrogen bond lifetime between N-H..O in a
+polyamide system.  This will use the default geometric definition for hydrogen
+bonds of length 3.0 :math:`\\A` and angle 130 degrees.
+It will observe a window of 2.0 ps (`sample_time`) and try to gather 1000
+sample point within this time window (this relies upon the trajectory being
+sampled frequently enough).  This process is repeated for 20 different start
+points to build a better average.
+
 ::
 
+  import MDAnalysis as mda
   from MDAnalysis.analysis import hbonds
+  from MDAnalysis.tests.datafiles import TRZ_psf, TRZ
   import matplotlib.pyplot as plt
+  # load system
+  u = mda.Universe(TRZ_psf, TRZ)
+  # select atoms of interest into AtomGroups
   H = u.select_atoms('name Hn')
-  O = u.select_atoms('name O')
   N = u.select_atoms('name N')
-  hb_ac = hbonds.HydrogenBondAutoCorrel(u, acceptors = u.atoms.O,
-              hydrogens = u.atoms.Hn, donors = u.atoms.N,bond_type='continuous',
-              sample_time = 2, nruns = 20, nsamples = 1000)
+  O = u.select_atoms('name O')
+  # create analysis object
+  hb_ac = hbonds.HydrogenBondAutoCorrel(u,
+              acceptors=O, hydrogens=H, donors=N,
+              bond_type='continuous',
+              sample_time=2.0, nsamples=1000, nruns=20)
+  # call run to gather results
   hb_ac.run()
+  # attempt to fit results to exponential equation
   hb_ac.solve()
+  # grab results from inside object
   tau = hb_ac.solution['tau']
   time = hb_ac.solution['time']
   results = hb_ac.solution['results']
   estimate = hb_ac.solution['estimate']
+  # plot to check!
```

Comment:
Include plot to show what it looks like.

Eg see Ramachandran page.

Reply:
the datafiles in testsuite aren't big enough to produce a good plot (or even an awful plot).  You need to be saving frames really often to get this analysis done

Reply:
Can you use MDAnalysisData? I'd consider it fair game for examples. (We can put in a footnote or link on how to install it.)

Reply:
Yeah I could, but I still don't have a trajectory to hand for this

Reply:
Ok, then leave it for "later".